ddtrace/ext/peer.go

```diff
@@ -0,0 +1,14 @@
+package ext
+
+const (
+	// PeerHostIPV4 records IPv4 host address of the peer.
+	PeerHostIPV4 = "peer.ipv4"
+	// PeerHostIPV6 records the IPv6 host address of the peer.
+	PeerHostIPV6 = "peer.ipv6"
+	// PeerService records the service name of the peer service.
```

Comment:
I've generally been leaving a space between documentation and the previous declaration. However, I don't have any strong feelings against one way over the other.